saplings/circuits/src/components/forms/MultiStepForm.scss

```diff
@@ -144,7 +144,7 @@
     align-items: center;
     justify-content: space-between;
 
-    form {
+    .form {
```

Comment:
It would probably be better to either omit the form classname and just use the element selector, or give the element a specific class.

Reply:
I gave it a specific class name